infra/tf/configs/provider_node.yml

```diff
@@ -0,0 +1,31 @@
+#cloud-config
+groups:
+  - ubuntu: [root,sys]
+
+runcmd:
```

Comment:
could we use `get.fleek.network` script instead

Reply:
It's inconsistent for automation purposes in it's current state, I wanted to use it but ran into some issues and at least atm, it would require updating inputs the cloud-init gives if it changes at all. 

If we add like a `--noninteractive` mode and a way to pass in the domain and stuff with no user input, it could be used.

For bootstrap nodes though, we'd need a way to skip nginx steps in the assisted installer

Reply:
cc @heldrida

Reply:
Is this a blocker for the pr?